teams/habitflow.md

```diff
@@ -0,0 +1,10 @@
+# HabitFlow
+
+## Developers
+
+* Drew L. Mortenson, [DrewsCodeLife](https://github.com/DrewsCodeLife)
+* Halin Gailey, [hgailey](https://github.com/hgailey)
+* Nick Kaplan, [NickK21](https://github.com/NickK21)
+* Patrick Guevarra, [pvguevarra](https://github.com/pvguevarra)
```

Comment:
The links shouldn't have spaces between the name and the url.

Bad: `[name] (url)`
Good: `[name](url)`

Reply:
ignore this comment, yours is good